lib/datasource/pypi/index.ts

```diff
@@ -208,36 +212,13 @@ async function getSimpleDependency(
 export async function getReleases({
   compatibility,
   lookupName,
-  registryUrls,
+  registryUrl,
 }: GetReleasesConfig): Promise<ReleaseResult | null> {
-  let hostUrls = ['https://pypi.org/pypi/'];
-  if (is.nonEmptyArray(registryUrls)) {
-    hostUrls = registryUrls;
-  }
-  if (process.env.PIP_INDEX_URL) {
-    hostUrls = [process.env.PIP_INDEX_URL];
-  }
-  let dep: ReleaseResult;
-  for (let index = 0; index < hostUrls.length && !dep; index += 1) {
-    let hostUrl = hostUrls[index];
-    hostUrl += hostUrl.endsWith('/') ? '' : '/';
-    if (hostUrl.endsWith('/simple/') || hostUrl.endsWith('/+simple/')) {
-      logger.trace(
-        { lookupName, hostUrl },
-        'Looking up pypi simple dependency'
-      );
-      dep = await getSimpleDependency(lookupName, hostUrl);
-    } else {
-      logger.trace({ lookupName, hostUrl }, 'Looking up pypi api dependency');
-      dep = await getDependency(lookupName, hostUrl, compatibility);
-    }
-    if (dep !== null) {
-      logger.trace({ lookupName, hostUrl }, 'Found pypi result');
-    }
+  const hostUrl = registryUrl + (registryUrl.endsWith('/') ? '' : '/');
```

Comment:
We have thousands of variants to add ending slash, we should ha a string util for that